src/webhook.ts

```
@@ -0,0 +1,47 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+import logger from '@/utils/logger.js';
+import { Bot, webhookHandler } from 'gramio';
```

ESLint

[error] 'webhookHandler' is defined but never used.

(@typescript-eslint/no-unused-vars)